src/core/reducers/installations.js

```diff
@@ -13,10 +14,12 @@ import {
   UNINSTALL_COMPLETE,
   acceptedInstallTypes,
 } from 'core/constants';
+import log from 'core/logger';
 
 
 export default function installations(state = {}, { type, payload }) {
   if (!acceptedInstallTypes.includes(type)) {
+    log.info(`${type} not in acceptedInstallTypes so action is being ignored.`);
```

Comment:
I think this is so it doesn't do any work for unrelated actions. Since type could be anything like "CALL_API" or "SET_USER_TOKEN". Unfortunately since this is adding a new action it's a bit annoying.

Will this log on every non-installation specific action now?